docsite/docusaurus.config.ts

```diff
@@ -54,12 +55,13 @@ const config: Config = {
 
   themeConfig: {
     // Replace with your project's social card
-    image: 'img/docusaurus-social-card.jpg',
+    image: 'img/rnmp_logo.png',
     navbar: {
       title: 'React Native MediaPipe',
       logo: {
         alt: 'My Site Logo',
-        src: 'img/logo.svg',
+        // src: 'img/logo.svg',
```

Comment:
Can you please a) delete the commented out line as opposed to commenting it out (if you need the original, you can always check in the history) and b) delete the logo.svg file in order not to just pile up crud ?